lib/gst/player/gstplayer.c

```diff
@@ -1796,6 +1926,7 @@ gst_player_media_info_create (GstPlayer * self)
   GST_DEBUG_OBJECT (self, "begin");
   media_info = gst_player_media_info_new (self->uri);
   media_info->duration = gst_player_get_duration (self);
+  media_info->tags  = self->global_tags;
```

Comment:
And set self->global_tags to NULL. Otherwise you might free them twice.

Also unref global_tags in stop_internal(), finalize() and emit_error()